reader_test.go

```diff
@@ -2,12 +2,35 @@ package ndjson
 
 import (
 	"bytes"
+	"fmt"
 	"strings"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 )
 
+func text(size int) []byte {
+	letters := []byte("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+	buf := make([]byte, size)
+	for i := range buf {
+		buf[i] = letters[i%len(letters)]
+	}
+	return buf
+}
+
+func docs(sizes ...int) []byte {
+	var buf []byte
+	for i := 0; i < len(sizes); i++ {
+		if i > 0 {
+			buf = append(buf, '\n')
+		}
+		buf = append(buf, fmt.Sprintf(`{"id":%d,"text":"`, i+1)...)
+		buf = append(buf, text(sizes[i])...)
+		buf = append(buf, `"}`...)
+	}
+	return buf
+}
```

Comment:
I restrained myself from extracting the `text` and `docs` functions and the `Doc` struct (now duplicated in 3 locations) in a separate file named `fixture_test.go`. Let me know if I should proceed.

Reply:
Ah, no worries. I like it :-)